VideoWeb/VideoWeb.Common/Models/Conference.cs

```diff
@@ -52,5 +54,9 @@ private CivilianRoom GetOrCreateCivilianRoom(long roomId)
 
             return room;
         }
+        public bool IsConferenceInSession()
```

Comment:
```suggestion
        public boolean IsInSession => CurrentStatus == ConferenceState.InSession;
```

Reply:
This is no need any more I will remove this line